hugegraph-core/src/main/java/com/baidu/hugegraph/backend/serializer/SerializerFactory.java

```diff
@@ -49,7 +52,8 @@ public static AbstractSerializer serializer(String name) {
 
         assert AbstractSerializer.class.isAssignableFrom(clazz);
         try {
-            return clazz.getConstructor().newInstance();
+            Constructor<?>[] constructors = clazz.getConstructors();
+            return (AbstractSerializer) constructors[0].newInstance(configuration);
```

Comment:
can keep all the class existing only one constructor?

Reply:
clazz.getConstructor(HugeConfig.class).newInstance(config);